app/assets/javascripts/discourse/app/components/composer-editor.js

```diff
@@ -859,15 +871,6 @@ export default class ComposerEditor extends Component {
 
   @action
   extraButtons(toolbar) {
-    toolbar.addButton({
```

Comment:
We're deliberately dropping this button for the old composer?

Reply:
Yes, this was also done in the toolbar PR, it's a deliberate removal